src/Root.tsx

```diff
@@ -0,0 +1,17 @@
+import { HashRouter as Router, Routes, Route } from 'react-router-dom';
+import { HomePage } from './pages/HomePage';
+import { PeoplePage } from './pages/PeoplePage';
+import { NotFoundPage } from './pages/NotFoundPage';
+import { App } from './App';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="people/:slug?" element={<PeoplePage />} />
```

Comment:
Divide these routes and make attachments. So the routing will look more structured.